_data/devices/sirius.yml

```diff
@@ -0,0 +1,65 @@
+architecture:
+  cpu: arm64
+  userspace: arm
+battery:
+- A0001:
+    capacity: 3120
+    removable: false
+    tech: Li-Po
+bluetooth:
+  profiles:
+  - A2DP + aptX HD
+  spec: '5'
+cameras:
+- flash: LED
+  info: 12 MP
+- flash: LED
+  info: 5 MP
+- flash: ''
+  info: 5 MP
+codename: sirius
+cpu: Kryo 360
+cpu_cores: '8'
+cpu_freq: 2 x 2.2 GHz + 6 x 1.7 GHz
+depth: 7.5 mm (0.30 in)
+download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Power</kbd>.
+  Keep holding both buttons until the word "FASTBOOT" appears on the screen, then
+  release.
+gpu: Adreno 616
+height: 147.3 mm (5.80 in)
+install_method: fastboot_xiaomi
+maintainers:
+- solidv
+name: Mi 8 SE
+network:
+- 2G GSM
+- 2G CDMA
+- 3G UMTS
+- 4G LTE
+peripherals:
+- Dual SIM
+- Fingerprint reader
+- Accelerometer
+- Barometer
+- Gyroscope
+- Hall effect sensor
+- Light sensor
+- Proximity sensor
+- Compass
+- IR
```

Comment:
By IR you mean IR blaster? if yes, change it

Reply:
He means IR for faceunlock. Sirius also has it so does beryllium and dipper

Reply:
so it's IR camera

Reply:
We dont have IR camera, we have IR blaster